modules/nw-operator-cr.adoc

```diff
@@ -347,6 +347,10 @@ defaultNetwork:
     genevePort: 6081
     ipsecConfig: {}
 ----
+[IMPORTANT]
+====
+Using `OVNKubernetes` can lead to stack exhaustion problem on {ibm-power-name}.
```

Comment:
```suggestion
Using OVN-Kubernetes can lead to stack exhaustion problem on {ibm-power-name}.
```

Reply:
I've seen the backticks for when it is a literal value, but never for a general reference. However, I then started looking around and saw that most of the install docs use the backticks so I'm not sure. I write for OVN-K and now you have me wondering if we need to address this in the style guide.

Reply:
@JoeAldinger Thank you the suggestion. I agree. Updated the PR accordingly.